data/human/human missions.txt

```diff
@@ -3601,3 +3601,79 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
+	# Don't require the player to pay credits again if the player was already allowed to land when this mission triggered.
+	clearance
+	to offer
+		random < 8
+		credits >= 2000000
+	on offer
+		conversation
+			"You see a warning that one of the repulsors on your ship is not working when you start de-orbiting."
```

Comment:
```suggestion
			When you start de-orbiting, you see a warning that one of the repulsors on your ship is not working."
```
Personal preference - I just think it flows a little better this way. Up to you!